baseapp/streaming.go

```diff
@@ -0,0 +1,30 @@
+package baseapp
+
+import (
+	"sync"
+
+	abci "github.com/tendermint/tendermint/abci/types"
+
+	store "github.com/cosmos/cosmos-sdk/store/types"
+	"github.com/cosmos/cosmos-sdk/types"
+)
+
+// Hook interface used to hook into the ABCI message processing of the BaseApp
+type Hook interface {
```

Comment:
see above comment. This type name is just too broad. Perhaps `StreamingListener`?

Reply:
That sounds good to me, thanks!